source/compose.manager/compose.manager.settings.page

```diff
@@ -5,27 +5,10 @@ Type="xmenu"
 Menu="Utilities"
```

Comment:
You have removed a bunch of stuff from this file that ought not to be removed.

Reply:
Yes because the files in the source folder are not reflecting what’s installed when the plugin is installed on Unraid when it‘s installed, at least not on 6.11.0-rc5p2

Reply:
Yeah thats because i dont have a dev branch in this repo. The latest release is 75480f1c55ef48e54aa1c2aa2fc01c21b41dc98d, the main branch has several not yet released changes.

Reply:
Understood, but do you even consider to implement this anyways?

Reply:
Yes I am considering implementing this. Though I will probably have the default be putting compose under the docker tab. I would also want to do a little homework and see if other plugins implement this feature in the same way (unless you already have).

Reply:
I close this for now and open a new one, TBH it was really late yesterday and I even didn't notice that I've misspelled "choose" in the tilte...